python/tests/tsutil.py

```diff
@@ -333,8 +333,8 @@ def generate_site_mutations(tree, position, mu, site_table, mutation_table,
     more than one mutation per edge.
     """
     assert tree.interval[0] <= position < tree.interval[1]
-    states = {"A", "C", "G", "T"}
-    state = random.choice(list(states))
+    states = ["A", "C", "G", "T"]
+    state = random.choice(states)
```

Comment:
The tests were not deterministic because of different orderings of sets, btw. This fixes that.

Reply:
Dammit, got caught by that one before!